### DISCUSSION TITLE
Enable use of {@pod.*} in pick where options

### DISCUSSION THREAD

**Discussion by pcfreak30** [pods-framework/pods#6763] [Ideas]
This is adding 1 line of code to pick.php around line 2043 `if ( ! empty( $params['where'] ) && (array) $options['table_info']['where_default'] !== $params['where'] ) {`

since the `@pods` magic tag assumes a global, setting `$GLOBALS['pods'] = $pod;` allows this to work.

However since im not a fan of blindly trampling over global variables I think `pods_v` should check for another source for the pods object.

My use case has been `exchange.ID is null or exchange.ID = {@pods.ID}` to only allow a related item in a relationship to show that matches the current pod item or is not linked to one to prevent "in use" items from showing up in a list. Overall this is highly useful for dynamic where queries with "introspection" without the use of php filters or tag filters.

**Comment by quasel** [comment]
why not create an PR for it ;) sounds reasonable - but maybe @sc0ttkclark has objections ^^ 

i solved a issue like that with a custom where `relationship_field.ID IN( "{@get.post}", "" )` so not sure if this is needed ;) Thoughts?
